images/ubuntu/templates/ubuntu-24.04.pkr.hcl

```diff
@@ -188,6 +188,11 @@ build {
     script          = "${path.root}/../scripts/build/configure-apt-mock.sh"
   }
 
+  provisioner "file" {
+    destination = "${var.helper_script_folder}"
+    source      = "${path.root}/../scripts/helpers"
+  }
+
```

Comment:
Why did you move this section? Does this change anything?

Reply:
i feel packer will execute  in sequence manner, so before execution of configure-apt.sh , we have to copy the file(os.sh) which are available in HELPER folder in git path to $HELPER_FOLDER